src/metatrain/experimental/nanopet/modules/samples.py

```diff
@@ -0,0 +1,535 @@
+from typing import Dict, List, Tuple
+
+import torch
+from metatensor.torch import Labels
+from metatensor.torch.atomistic import System
+
+
+def get_samples(
```

Comment:
documentation please!

Reply:
I was following suit on what is in `concatenate_structures` in structure.py in the same directory! :D But sure I can add